R/estimate_cuminc.R

```diff
@@ -7,44 +7,40 @@
 #' Column must be a factor and the first level indicates censoring, the
 #' next level is the outcome of interest, and the remaining levels are the
 #' competing events. Default is `"CNSR"`
+#' @param data The name of the dataset for Time-to-Event analysis based on the Analysis Data Model (ADaM) principles. The dataset is expected to have
+#'   one record per subject per analysis parameter. Rows in which the analysis variable (AVAL) or the censor variable (CNSR) contain NA, are removed during analysis.
 #' @param conf.int Confidence internal level. Default is 0.95.
 #' @param ... Additional argument passed to `tidycmprsk::cuminc()`
-#' @inheritParams estimate_KM
+#' @inheritParams estimate_cuminc
```

Comment:
@SHAESEN2 

FYI this is inheriting the parameter descriptions from the `estimate_KM()` function documentation, since both functions share most arguments. This should remain as `@inheritParams estimate_KM`